src/NHSD.BuyingCatalogue.Identity.Api/Infrastructure/BackLinkTagHelper.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Text.Encodings.Web;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.TagHelpers;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.Infrastructure
+{
+    [HtmlTargetElement(TagHelperConstants.Anchor, Attributes = TagHelperName)]
+    public sealed class BackLinkTagHelper : AnchorTagHelper
+    {
+        public const string TagHelperName = "nhs-back-link";
+        public const string BackLinkLabel = "back-link-label";
+
+        public BackLinkTagHelper(IHtmlGenerator generator) : base(generator)
```

Comment:
Optional suggestion, I think having the constructor initializer on a new line (like below) is a bit better in terms of readability but will leave to you to decide.

```c#
public BackLinkTagHelper(IHtmlGenerator generator)
    : base(generator)
```

Reply:
👍